src/linear_algebra/basis.lean

```diff
@@ -200,6 +200,13 @@ end
 by simp only [basis.sum_coords, linear_map.id_coe, linear_equiv.coe_coe, id.def, basis.repr_self,
   function.comp_app, finsupp.coe_lsum, linear_map.coe_comp, finsupp.sum_single_index]
 
+lemma coord_dvd_of_dvd (i : ι) (m : M) (r : R) : r ∣ b.coord i (r • m) :=
```

Comment:
Since there is only one `dvd` in the statement, how about:
```suggestion
lemma dvd_coord_smul (i : ι) (m : M) (r : R) : r ∣ b.coord i (r • m) :=
```